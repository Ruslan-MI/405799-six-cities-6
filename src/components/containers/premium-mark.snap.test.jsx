import React from "react";
import {
  render,
} from "@testing-library/react";
import PremiumMark from "./premium-mark";


it(`Premium mark should render correctly`, () => {
  const {
    container,
  } = render(<PremiumMark />);

  expect(container).toMatchSnapshot();
});
