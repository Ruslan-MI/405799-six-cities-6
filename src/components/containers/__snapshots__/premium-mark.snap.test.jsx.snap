// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Premium mark should render correctly 1`] = `
<div>
  <div
    class="place-card__mark"
  >
    <span>
      Premium
    </span>
  </div>
</div>
`;
